examples/light-switch-app/efr32/BUILD.gn

```diff
@@ -63,6 +63,12 @@ declare_args() {
 
   # Argument to force enable WPA3 security
   rs91x_wpa3_only = false
+
+  #default WiFi SSID
+  ssid = ""
```

Comment:
to avoid conflicts with other gn flags, consider prefixing like `chip_example_ssid` or `chip_default_wifi_ssid` and simlarely for psk